hardpy/pytest_hardpy/utils/dialog_box.py

```diff
@@ -205,6 +207,71 @@ def convert_data(self, input_data: str | None = None) -> bool:
         return True
 
 
+class StepWidget(IWidget):
+    """Step widget.
+
+    Args:
+        title (str): Step title
+        text (str | None): Step text
+        widget (IWidget | None): Step widget
```

Comment:
Only `ImageWidget`, not `IWidget`. Behavior with other widgets will be incorrect if the value is returned.